graspologic/models/edge_swaps.py

```diff
@@ -87,9 +99,7 @@ def _do_setup(self) -> np.ndarray:
         edge_list = np.stack((row_inds, col_inds)).T
         return edge_list
 
-    def swap_edges(
-        self, n_swaps: int = 1, seed: Optional[int] = None
-    ) -> Tuple[AdjacencyMatrix, np.ndarray]:
+    def swap_edges(self, n_swaps: int = 1) -> Tuple[AdjacencyMatrix, np.ndarray]:
```

Comment:
Just to note, but if you're changing the default parameter here from None to 1, you'll need to increment the minor version at least.

Reply:
@daxpryce its more like this parameter was removed, but i am guessing your comment still applies. 

this whole class was just added a few days ago (so we dont have a release with it) - what does that mean for semver? I thought we were bumping versions etc. based on actual releases but i defer to you, semver protocol never seems to get through my thick skull